stdlib/email/base64mime.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Callable
+
+def header_length(bytearray: bytes) -> int: ...
+def header_encode(header_bytes: str | bytes, charset: str = ...) -> str: ...
+def body_encode(s: bytes, maxlinelen: int = ..., eol: str = ...) -> str: ...
```

Comment:
Can only be applied to `bytes`, due to the use of `b2a_base64`
![image](https://user-images.githubusercontent.com/80432516/147124301-74231f09-01e7-412b-a6fa-d5f427ab2fab.png)